{"name": "test_failed_login", "status": "passed", "start": 1741348535289, "stop": 1741348535603, "uuid": "ed40864b-6f04-448c-9f5d-3ba43c5bc052", "historyId": "06b4296db8b878c7965c434c2acf5e7d", "testCaseId": "06b4296db8b878c7965c434c2acf5e7d", "fullName": "tests.api.test_auth#test_failed_login", "labels": [{"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_auth"}, {"name": "host", "value": "DESKTOP-GIDTMH7"}, {"name": "thread", "value": "19652-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_auth"}]}